### PR TITLE
system/mailutils: Update for version 3.17

### PR DESCRIPTION
**Pull request by graygnuorg**

### CODE REVIEW

File: system/mailutils/mailutils.SlackBuild [hunk 1]
```diff
@@ -25,7 +25,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=mailutils
-VERSION=${VERSION:-3.11.1}
+VERSION=${VERSION:-3.17}
 BUILD=${BUILD:-2}
```
Comment (willysr) [inline system/mailutils/mailutils.SlackBuild:29]: this should be reset to 1 when you bump VERSION

**Review by willysr — CHANGES_REQUESTED** [review]
please fix this

**Comment by willysr** [comment]
merged

**Comment by willysr** [comment]
@sbo-bot: build system/mailutils

**Comment by willysr** [comment]
merged

**Comment by graygnuorg** [comment]
Fixed. Thank you.

**Pull request closed without merging**